cmake/Utils.cmake

```diff
@@ -264,10 +266,22 @@ endfunction()
 #
 function(caffe2_include_directories)
   foreach(path IN LISTS ARGN)
-    if (${path} MATCHES "/anaconda")
-      include_directories(AFTER SYSTEM ${path})
+    if (${DEPRIORITIZE_ANACONDA})
```

Comment:
So I think we should probably going to enforce "if conda, only conda" to simplify this chunk, but I guess it is fine for now. Mixing system headers and conda headers is a bit error prone.

Reply:
Totally agree.